lib/usd/ui/layerEditor/saveLayersDialog.cpp

```diff
@@ -227,6 +262,32 @@ void SaveLayerPathRow::onTextChanged(const QString& text)
         _relativeAnchor = MayaUsd::utils::getSceneFolder();
         _absolutePath = (_relativeAnchor / inputPath).lexically_normal();
     }
+
+    postUpdate();
+}
+
+void SaveLayerPathRow::postUpdate()
+{
+    // Update _pathEdit tooltip
+    QString tooltip;
+    if (!_relativeAnchor.empty()) {
+        tooltip = "Directory: ";
+        tooltip += _relativeAnchor.generic_string().c_str();
+    }
+    _pathEdit->setToolTip(tooltip);
+
+    // Update relative anchors of child layers
+    _parent->forEachEntry([this](QWidget* w) {
+        auto entry = dynamic_cast<SaveLayerPathRow*>(w);
+        if (entry && entry->needToSaveAsRelative()
+            && (entry->_layerInfo.parent._layerParent == _layerInfo.layer)) {
+            ghc::filesystem::path relativeAnchor
```

Comment:
@pierrebai-adsk not sure I understand what you mean exactly, but if you are talking about multiple-level inheritance, this will work because: Here we process our immediate children and call "entry->setPathToSaveAs" and inside there, there is a call to postUpdate which will process immediate children of our child and so on recursion will go, if needed.